apps/design-system/content/docs/icons.mdx

```diff
@@ -55,6 +55,7 @@ Follow these steps to add a new custom icon to the Supabase icon library.
 
    - Exported at 24x24px with `viewBox="0 0 24 24"`
    - Uses `stroke="currentColor"` for strokes (no hardcoded colors)
+   - Uses `stroke-width="1.5"` unless there is a good reason to deviate
```

Comment:
Lucide tends to have a `stroke-width` of `2` when exporting an SVG from their site, whereas our versions tend to use `1.5`. Better to align, I think.

Reply:
I believe this is overridden by the prop so not a huge deal.